compiler/rustc_attr_parsing/src/attributes/deprecation.rs

```diff
@@ -13,16 +12,13 @@ pub(crate) struct DeprecationParser;
 
 fn get(
     cx: &AcceptContext<'_>,
-    ident: Ident,
+    name: Symbol,
```

Comment:
it surprised me we never use the span of this ident. But I guess we usually use the param_span. Well, nice!